From 973ab210298615d43417d3d191ec046eccce10e9 Mon Sep 17 00:00:00 2001
From: haykam821 <24855774+haykam821@users.noreply.github.com>
Date: Thu, 2 Sep 2021 07:35:48 -0400
Subject: [PATCH] Fix the player parameter of methods used for playing
 sounds in the World class being unclear (#2646)

* Fix the player parameter of methods used for playing sounds in the World class being unclear

* Document the except parameter of methods used for playing sounds in the World class
---
 mappings/net/minecraft/world/World.mapping | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/mappings/net/minecraft/world/World.mapping b/mappings/net/minecraft/world/World.mapping
index 2d67831b4f..be2298d9d1 100644
--- a/mappings/net/minecraft/world/World.mapping
+++ b/mappings/net/minecraft/world/World.mapping
@@ -130,7 +130,8 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 	METHOD method_8442 getSkyAngleRadians (F)F
 		ARG 1 tickDelta
 	METHOD method_8449 playSoundFromEntity (Lnet/minecraft/class_1657;Lnet/minecraft/class_1297;Lnet/minecraft/class_3414;Lnet/minecraft/class_3419;FF)V
-		ARG 1 player
+		ARG 1 except
+			COMMENT the player that should not receive the sound, or {@code null}
 		ARG 2 entity
 		ARG 3 sound
 		ARG 4 category
@@ -158,7 +159,8 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 		ARG 1 pos
 		ARG 2 direction
 	METHOD method_8465 playSound (Lnet/minecraft/class_1657;DDDLnet/minecraft/class_3414;Lnet/minecraft/class_3419;FF)V
-		ARG 1 player
+		ARG 1 except
+			COMMENT the player that should not receive the sound, or {@code null}
 		ARG 2 x
 		ARG 4 y
 		ARG 6 z
-- 
2.32.0.windows.2

